ci/kokoro/macos/build-bazel.sh

```diff
@@ -19,9 +19,7 @@ source "$(dirname "$0")/../../lib/init.sh"
 source module /ci/etc/integration-tests-config.sh
 source module /ci/lib/io.sh
 
-echo
-echo "================================================================"
-io::log_yellow "update or install Bazel."
+io::log_h2 "update or install Bazel."
```

Comment:
I guess I'd remove all the trailing periods from things that are now in boxes.

Reply:
done